tests/ui_tests/test_ui_add_component_window.py

```diff
@@ -1338,6 +1338,172 @@ def test_UI_GIVEN_component_with_cylinder_shape_WHEN_editing_component_THEN_cyli
             assert dialog.cylinderOptionsBox.isEnabled()
 
 
+def test_UI_GIVEN_component_with_scalar_field_WHEN_editing_component_THEN_field_appears_in_fields_list_with_correct_value(
+    qtbot
+):
+    instrument = Instrument(NexusWrapper("test_component_editing_scalar_field"))
+    model = ComponentTreeModel(instrument)
+    component_name = "chopper1"
+    component = instrument.create_component(component_name, "NXdisk_chopper", "")
```

Comment:
These lines are basically the same in all the tests, can we just put into a helper method?